src/lib/Gateway/DoctrineSchemaMetadataRegistry.php

```
@@ -50,13 +50,16 @@ private function ensureInitialized(): void
             $metadata = $gateway->getMetadata();
 
             $tableName = $metadata->getTableName();
-            if (isset($this->tableToMetadata[$tableName])) {
+            if (isset($this->tableToMetadata[$tableName]) && $this->tableToMetadata[$tableName] != $metadata) {
```

To put it simply, if two metadata services exist for the same table (even if they have the exact same properties), that is an error and shall not be ignored :)

The error should be found and removed, not silently ignored.

---

@Steveb-p so we need to fix it in https://github.com/ibexa/product-catalog/ somehow ? Should i close this PR ?